Serenity updates - October 2020 - Titles and structure (#783)
akeneo/pim-helpcenter: content/updates/2020-10/3-A-better-product-grid-for-product-models-and-variants.md

---
pim_announcement_img: ./img/.png
pim_announcement_alt_img: ""
pim_announcement_audience:
- CE
- EE
---

> # A better product grid for product models and variants

Reviewer: ```suggestion
# Product grid improvements for product models and variants
```

Author: ```suggestion
# Better manage product models and variants in the grid
```
